README.md

```diff
@@ -55,6 +55,75 @@ depending on processor support and the target execution environment
 (*multiplexing*). Furthermore, EverCrypt offers an (*agile*) API that makes it
 simple to switch between algorithms (e.g., from SHA2 to SHA3).
 
+## Citing HACL*
+
+If you want to cite HACL\* in a research publication, here is some guidance to
+make sure you pick the right publications.
+
+- For HACL* in its current incarnation, including SIMD versions of algorithms: CCS'20.
+
+```
+@inproceedings{polubelova2020haclxn,
+  title={Haclxn: Verified generic SIMD crypto (for all your favourite platforms)},
```

Comment:
title = {{HACLxN}: Verified generic {SIMD} crypto (for all your favourite platforms)}